decidim-core/app/controllers/decidim/participatory_processes_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require_dependency "decidim/application_controller"
+
+module Decidim
+  # A controller that holds the logic to show ParticipatoryProcesses in a
+  # public layout.
+  class ParticipatoryProcessesController < ApplicationController
+    helper_method :participatory_processes, :participatory_process, :hero_processes
+
+    private
+
+    def participatory_process
+      @participatory_process ||= participatory_processes.find(params[:id])
+    end
+
+    def participatory_processes
+      @participatory_processes ||= current_organization.participatory_processes
+    end
+
+    def hero_processes
```

Comment:
I'd like `promoted_processes` instead, it's more semantic - the fact that it's displayed as "hero" is a design concern.